crates/forge/src/main.rs

```diff
@@ -140,6 +145,7 @@ fn combine_configs(
         fuzzer_seed
             .or(forge_config.fuzzer_seed)
             .unwrap_or_else(|| thread_rng().next_u64()),
+        save_call_trace || forge_config.save_call_trace,
```

Comment:
I don't think there should be this logical operator here, it might lead to confusion.
Why is the `save_call_trace` a mandatory parameter as well?

Reply:
It is a boolean (flag), defaulted to false.

Reply:
Yup, it is the same as for `exit_first`